src/utils/HttpClient.js

```diff
@@ -1,75 +0,0 @@
-const https = require('https');
```

Comment:
This file was entirely unused so I removed it. Seems like you guys went with axios which is totally fine so you don't need this file.